ethcore/src/engines/authority_round/mod.rs

```diff
@@ -1114,8 +1114,10 @@ impl Engine<EthereumMachine> for AuthorityRound {
 
 		match verify_timestamp(&self.step.inner, header_step(header, self.empty_steps_transition)?) {
 			Err(BlockError::InvalidSeal) => {
-				let set_number = self.epoch_set(header)?.1;
-				self.validators.report_benign(header.author(), set_number, header.number());
+				if let Ok((_, set_number)) = self.epoch_set(header) {
```

Comment:
Maybe add a comment *why* it may fail and why it's ok?

Reply:
Done.